src/plays/flip-card-effect/FlipCardEffect.jsx

```diff
@@ -0,0 +1,45 @@
+import { getPlayById } from "meta/play-meta-util";
+
+import PlayHeader from "common/playlists/PlayHeader";
+import Card from "./Card";
+import "./FlipCardEffect.css";
+
+function FlipCardEffect(props) {
+  // Do not remove the below lines.
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+  const quotes = [
+    "Spread love everywhere you go.",
+    "if you love someone give them infinite space.",
+    "The happiest people are the givers not the takers.",
+    "The journey of a thousand miles begins with one step.",
+  ];
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={play} />
+        <div className="play-details-body">
+          {/* Your Code Starts Here */}
+          <div className="flip-card">
+            <h1 className="heading">Flip Card Effect</h1>
+            <h3 className="sub-heading">Click any card to flip</h3>
+            <div className="container">
+              {quotes.map((item, index) => {
```

Comment:
This can be a single string in a paragraph right? I mean why to keep it in an array data structure and loop?

Reply:
Ignore this comment.